src/stories/NetworkMonitor/ResponseBody.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ComponentProps } from "react";
+import { Story, Meta } from "@storybook/react";
+import ResponseBody from "ui/components/NetworkMonitor/ResponseBody";
+import { requestSummary } from "./utils";
+
+export default {
+  title: "Network Monitor/Response Body",
+  component: ResponseBody,
+} as Meta;
+
+const Template: Story<ComponentProps<typeof ResponseBody>> = args => <ResponseBody {...args} />;
+
+export const Basic = Template.bind({});
+
+const request = requestSummary("1", "https://app.replay.io/graphql", 200);
+
+Basic.args = {
+  request,
+  responseBodyParts: [
+    {
+      length: 12,
+      offset: 0,
+      value: "Zm9vYmFyCg==",
+    },
```

Comment:
Seems like a test with multiple parts is in order.

Reply:
Done!